.github/actions/javascript/authorChecklist/authorChecklist.js

```diff
@@ -0,0 +1,78 @@
+const core = require('@actions/core');
+const github = require('@actions/github');
+const https = require('https');
```

Comment:
Why not just use the native JS `fetch` instead of this node library? Does that simplify things where you don't need to do this stuff?

```js
res.on('data', chunk => fileContents += chunk);
res.on('end', () => ...);
```

and instead you could just do:

```js
function getNumberOfItemsFromAuthorChecklist() {
    return fetch(pathToChecklist)
        .then(res => res.text())
        .then((fileContents) => {
            // Do stuff
        });
}
```

Reply:
Oh yeah, I guess fetch would work. My brain was stuck in node.js land

Reply:
Hm, no... `fetch` isn't included in mainline node.js yet :( it's only included in experimental node.js 17 since earlier this year.

Reply:
Oh, gotcha. TIL